### DISCUSSION TITLE
How to open/close the "table of contents " in text note?

### DISCUSSION THREAD

**Discussion by zhang-stephen** [TriliumNext/Trilium#3352] [Q&A] [answered]
as the title said, the "table of contents" seems to be switched automatically. I want to switch it manually, thanks in advance.

**Comment by fooyoo** [comment, marked as answer]
add attribute: 
- toc - #toc or #toc=show will force the table of contents to be shown, #toc=hide will force hiding it.

https://github.com/zadam/trilium/wiki/Attributes

**Reply by zhang-stephen** [reply]
Thank you. It would be better if there is a button, or a key shortcut to do this in my opinion. 😄
